### PR TITLE
toggle log changes interval dynamically

### PR DESCRIPTION
**Pull request by Excodibur**
Improve UI config page as suggested here: https://github.com/ioBroker/ioBroker.influxdb/issues/165

### CODE REVIEW

**Comment by Apollon77** [comment]
Will check asap!

**Comment by Excodibur** [comment]
Pull request was extended to enable disabling of health checks for Influx 2 (Issue https://github.com/ioBroker/ioBroker.influxdb/issues/178)

File: main.js [hunk 1]
```diff
@@ -222,7 +222,7 @@ function stopPing(adapter) {
 }
 
 function ping(adapter) {
-    adapter._client.ping && adapter._client.ping(adapter.config.pingInterval - 1000 < 0 ? 1000 : adapter.config.pingInterval - 1000)
+    adapter._client.ping && adapter.config.pingserver && adapter._client.ping(adapter.config.pingInterval - 1000 < 0 ? 1000 : adapter.config.pingInterval - 1000)
```
Comment (GermanBluefox) [inline main.js:225]: As the pingserver is new option, I would add here `adapter.config.pingserver !== false`

File: main.js [hunk 2]
```diff
@@ -282,6 +282,8 @@ function connect(adapter) {
             break;
     }
 
+    if (!adapter.config.pingserver)
```
Comment (GermanBluefox) [inline main.js:285]: The same here: `adapter.config.pingserver === false`

**Comment by GermanBluefox** [comment]
LGTM

**Comment by Apollon77** [comment]
will check and release tomorrow

**Pull request merged**